spec/definitions/V1/EnterpriseUpdateV1.yaml

```diff
@@ -0,0 +1,50 @@
+type: object
+properties:
+  id
+  name:
+    title: Name
+    type: string
+    maxLength: 255
+    minLength: 1
+  short_code:
+    title: Short code
```

Comment:
A short 10-char representation of the enterprise. Field is optional, as system will auto-generate a value if none is provided

Reply:
@jerynmathew @ammar2168 @bmehendale @kzotter   All this can be part of description field which open api supports. We need to have both title and description for each filed. Title being short and description like the one mentioned by Jeryn for short code.